pkg/controller/rds/claim.go

```diff
@@ -21,6 +21,8 @@ import (
 	"fmt"
 	"strings"
 
+	aws "github.com/crossplaneio/stack-aws/pkg/clients"
+
```

Comment:
```go
resource.WithManagedBinder(resource.NewAPIManagedStatusBinder(mgr.GetClient())),
resource.WithManagedFinalizer(resource.NewAPIManagedStatusUnbinder(mgr.GetClient())),
```

It just occurred to me that since we're enabling the status subresource for the RDS controller we'll also need to pass the above arguments to `resource.NewClaimReconcile`. I'm hoping to make this behaviour the default soon given that it should be the default for any new controller we write, per https://github.com/crossplaneio/crossplane-runtime/issues/29.